<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TileDialog</class>
 <widget class="QDialog" name="TileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="rbDoorNone">
       <property name="text">
        <string>None</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbDoorL">
       <property name="text">
        <string>Left door</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbDoorR">
       <property name="text">
        <string>Right door</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbFlagSpawn">
       <property name="text">
        <string>Flag spawn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbPlayerSpawn">
       <property name="text">
        <string>Player spawn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbBotSpawn">
       <property name="text">
        <string>Bot spawn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbWeaponSpawn">
       <property name="text">
        <string>Weapon spawn</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item alignment="Qt::AlignBottom">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Bot spawn:</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QComboBox" name="cbBotSpawn"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item alignment="Qt::AlignTop">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Weapon spawn:</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QComboBox" name="cbWeaponSpawn"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
